web-app/js/portal/cart/WpsDownloadHandler.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.cart');
+
+Portal.cart.WpsDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
```

Comment:
Well the choices are there now on a separate branches, so take your pick tomorrow with @jkburges . Get healthy!